lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -494,7 +494,7 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
     |> Enum.filter(fn
       %{name: name} -> String.starts_with?(name, "hana_#{String.downcase(sid)}_site_lss_")
     end)
-    |> Enum.map(fn %{name: name} -> name |> String.split("_") |> Enum.at(-1) end)
+    |> Enum.map(fn %{name: name} -> name |> String.split("_site_lss_") |> Enum.at(-1) end)
```

Comment:
I guess we are using now instead `_site_lss_` as the string that we use to mark the split, wouldn't it make sense to define this as a constant better?

Reply:
well, we have this string parts spread all around the policy, and they are not used anywhere else.
i find it acceptable